src/components/dialog/Dialog.spec.jsx

```diff
@@ -160,4 +160,36 @@ describe('<Dialog>', () => {
 
     expect(closeBtn.props()['data-testid']).toEqual('test_id');
   });
+
+  it('fires onExitTransitionEnd callback after cleaning no-scroll class', () => {
+    const onExitTransitionEnd = () => {
+      expect(
+        document.body.classList.contains('sg-dialog-no-scroll')
+      ).toBeFalsy();
+    };
+
+    const wrapper = mount(
+      <Dialog onExitTransitionEnd={onExitTransitionEnd} open>
+        content text
+      </Dialog>
+    );
+
+    wrapper.setProps({open: false});
+  });
+
+  it('does not clean no-scroll class after onEntryTransitionEnd callback', () => {
```

Comment:
I wanted to express that it forces cleanup before the `EXIT` callback because it has become a business requirement, but not the `ENTRY` one. I had already a bug related to the second, during this implementation. Maybe it's a matter of rephrasing the description that it's about "forcing" cleanup.

Reply:
https://github.com/brainly/style-guide/pull/2388/commits/2cc5dcb4dda6d326b76f749b0f6844b74bc0b820

Reply:
Ok, thanks. Do you think it would be good to also test in what moment `no-scroll` class  is being added?

Reply:
I think, previous tests descriptions were better 😄  `bodyNoScroll` is implementation detail. Can we stay with no-scroll class because it's actually effect of running component?

Reply:
https://github.com/brainly/style-guide/pull/2388/commits/b64eb41964ae3fa415bd736e2441e63e16b4f5bb